src/js/utils/checkBrowserCompatibilty.js

```diff
@@ -1,8 +1,20 @@
+import { serial as serialPolyfill } from './web-serial-polyfill/serial.ts';
+
 export  function checkBrowserCompatibility() {
     const compatible = "serial" in navigator;
 
     if (!compatible) {
-        const errorMessage = "Betaflight app requires Chrome, Chromium, Edge or Vivaldi browser.";
+        if('usb' in navigator &&
+            !('brave' in navigator)
+        ) {
+            navigator.serial = serialPolyfill;
+        }
+
+        if(navigator.serial || navigator?.bluetooth){
```

Comment:
This will need a rebase as the file has been updated with `isChromium()` function